src/txorphanage.h

```diff
@@ -29,11 +29,11 @@ class TxOrphanage {
     /** Extract a transaction from a peer's work set
      *  Returns nullptr and sets more to false if there are no transactions
      *  to work on. Otherwise returns the transaction reference, removes
-     *  the transaction from the work set, and populates its arguments with
-     *  the originating peer, and whether there are more orphans for this peer
+     *  the transaction from the work set, and populates "more" with
```

Comment:
nit: does "with and whether" make any sense? Not for me, so might be a typo.

Reply:
fixed